E2E-tests/config/api_config.py

```diff
@@ -116,6 +116,11 @@ class NodesApiConfig:
     selected_node: str = MISSING
     node: Node = MISSING
     token_policy_id: str = MISSING
+    permissioned_min_ariadne_d_param_value: int = MISSING
+    permissioned_max_ariadne_d_param_value: int = MISSING
+    registered_min_ariadne_d_param_value: int = MISSING
+    registered_max_ariadne_d_param_value: int = MISSING
```

Comment:
could we make it a little simpler in configuration?
```
    d_param_min: DParam = MISSING
    d_param_max: DParam = MISSING
```
and then in config:
```
        "d_param_min": {
            "permissioned_candidates_number": 1,
            "trustless_candidates_number": 1
        },
        "d_param_max": {
            "permissioned_candidates_number": 5,
            "trustless_candidates_number": 5
        }
```
and use it in tests like this:
```
p_floor = config.nodes_config.d_param_min.permissioned_candidates_number
```